Source/spells.cpp

```diff
@@ -196,28 +196,31 @@ void EnsureValidReadiedSpell(PlayerStruct &player)
 	}
 }
 
-bool CheckSpell(int id, spell_id sn, spell_type st, bool manaonly)
+SpellCheckResult CheckSpell(int id, spell_id sn, spell_type st, bool manaonly)
 {
 #ifdef _DEBUG
 	if (debug_mode_key_inverted_v)
-		return true;
+		return SpellCheckResult::Success;
 #endif
 
 	if (!manaonly && pcurs != CURSOR_HAND) {
-		return false;
+		return SpellCheckResult::Fail_Busy;

```

Comment:
You could probably trigger this (if you wanted to test it), by firing two fireballs and before the second on fire pick up an item in the inventory.

Reply:
in this case, maybe nothing should be said. so the buisy case should result in nothing